core/controllers/creator_dashboard.py

```diff
@@ -264,20 +248,20 @@ class NotificationsHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    @acl_decorators.can_access_creator_dashboard
```

Comment:
Yes.

The creator dashboard is not accessible to logged out user. So, that should not be a problem. (This should only show when you can access creator dashboard, right?)